modules/schain.js

```diff
@@ -1,207 +0,0 @@
-import {config} from '../src/config.js';
```

Comment:
This is a breaking change. Could be OK if we target 10 (current base is master), but it may be better to leave in some logic that keeps accepting the current `setConfig({schain})` and just moves it in the right place under ortb2 to ease the transition.